frontend/src/components/account/AccountPage.js

```diff
@@ -248,7 +260,34 @@ export default function AccountPage({
               </div>
             );
           } else if (i.type === "hubs") {
-            return <MiniHubPreviews hubs={i.value} />;
+            if (i.value.length <= 2) return <MiniHubPreviews hubs={i.value} />;
+            else {
+              const hubsToDisplay = showMoreSectors ? i.value : i.value.slice(0, 2);
+              return (
+                <>
+                  <div className={classes.subtitle}>
+                    {" "}
+                    {organizationTexts.organization_is_active_in_these_sectors}{" "}
+                  </div>
+                    <MiniHubPreviews hubs={hubsToDisplay} />
+                    <div className={classes.hubButtonsContainer}>
+                      <Button onClick={handleShowMoreSectors}>
+                        {showMoreSectors ? (
+                          <>
+                            <ExpandLessIcon />
+                            {texts.show_less}
+                          </>
+                        ) : (
+                          <>
+                            <ExpandMoreIcon />
+                            {texts.show_more}
+                          </>
+                        )}
+                      </Button>
+                    </div> 
```

Comment:
Maybe this could be incorporated into the MiniHubPreviews component. Then we could just add a `maxHubsToShow` prop and the complexity would be hidden and it could be reused.